src/models/Token/integration.spec.js

```diff
@@ -0,0 +1,59 @@
+import resetDatabase from '@tests/resetDatabase'
+import database from '@database/index'
+import { createToken, retrieveToken, validateToken } from './index'
+
+describe('The Token model helpers', () => {
```

Comment:
I know integration tests can’t always be split to separate files, but I don’t see why you have to split them here?
`createToken`, `retrieveToken` and `validateToken` could all be in their own file.